runit-focal/set_metrics_dir

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
None of the metrics dir stuff should be needed any more, but that may make more sense to leave for balboa cleanup folks to get to.